tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "resolveJsonModule": true,
     "declaration": true,
     "declarationDir": "./dist/types",
+    "skipLibCheck": true,
```

Comment:
This option skips type *validation* for packages in node_modules when building (the types still load though). Had to turn this on after adding the 0x/utils package because `tsc` complained about types conflicts btw mocha and jest as below. Think something's amiss in one of 0x's transitive dependencies.

```
node_modules/@types/jest/index.d.ts:34:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'beforeEach' must be of type 'HookFunction', but here has type 'Lifecycle'.

34 declare var beforeEach: jest.Lifecycle;
               ~~~~~~~~~~

  node_modules/@types/mocha/index.d.ts:2680:13
    2680 declare var beforeEach: Mocha.HookFunction;
                     ~~~~~~~~~~
    'beforeEach' was also declared here.

node_modules/@types/jest/index.d.ts:36:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'afterEach' must be of type 'HookFunction', but here has type 'Lifecycle'.

36 declare var afterEach: jest.Lifecycle;
               ~~~~~~~~~

  node_modules/@types/mocha/index.d.ts:2698:13
    2698 declare var afterEach: Mocha.HookFunction;
                     ~~~~~~~~~
    'afterEach' was also declared here.

node_modules/@types/jest/index.d.ts:37:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'describe' must be of type 'SuiteFunction', but here has type 'Describe'.

37 declare var describe: jest.Describe;
               ~~~~~~~~
```